css/styles.layout.css

```
@@ -2791,107 +2790,340 @@
+        border: 1px dashed rgba(var(--color-white-rgb), var(--alpha-16));
       }
 
-      .rerun-ranking-avatar-shell {
-        width: clamp(64px, 10vw, 108px);
-        height: clamp(64px, 10vw, 108px);
-        flex: 0 0 clamp(64px, 10vw, 108px);
-        border-radius: 12px;
+      /* --- Timeline Body --- */
+      .rerun-timeline-body {
+        --rerun-timeline-row-height: 52px;
+        display: flex;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)